templates/repo/settings/webhook/settings.tmpl

```diff
@@ -248,6 +248,15 @@
 	<span class="help">{{.locale.Tr "repo.settings.branch_filter_desc" | Str2html}}</span>
 </div>
 
+<!-- Authorization Header -->
+<div class="field{{if eq .HookType "matrix"}} required{{end}}">
+	<label for="authorization_header">{{.locale.Tr "repo.settings.authorization_header"}}</label>
+	<input id="authorization_header" name="authorization_header" type="text" value="{{.Webhook.HeaderAuthorization}}"{{if eq .HookType "matrix"}} placeholder="Bearer $access_token" required{{end}}>
```

Comment:
Should the placeholder also be shown when Hooktype isn't matrix?

Reply:
> Should the placeholder also be shown when Hooktype isn't matrix?

For matrix, it's required, for others, it could also be used but optional.